pkg/storaged/lvol-tabs.jsx

```diff
@@ -372,6 +505,117 @@
     });
 }
 
+const StructureDescription = ({ client, lvol }) => {
+    const struct = lvol.Structure;
+
+    if (!struct)
+        return null;
+
+    let status = null;
+    const status_code = client.lvols_status[lvol.path];
+    if (status_code == "partial") {
+        status = _("This logical volume has lost some of its physical volumes and can no longer be used. You need to delete it and create a new one to take its place.");
+    } else if (status_code == "degraded") {
+        status = _("This logical volume has lost some of its physical volumes but has not lost any data yet. You should repair it to restore its original redundancy.");
+    } else if (status_code == "degraded-maybe-partial") {
+        status = _("This logical volume has lost some of its physical volumes but might not have lost any data yet. You might be able to repair it.");
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20230531-081909-df2e1047-fedora-testing-daily//Coverage/lcov/github-pr.diff.gcov.html)